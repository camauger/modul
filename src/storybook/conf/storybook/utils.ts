export const modulComponentsHierarchyRootSeparator: string = 'modul-components|';

