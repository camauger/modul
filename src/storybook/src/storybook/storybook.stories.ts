import { Welcome } from './welcome/welcome';

export default {
    title: `storybook`
};

export const welcome: any = () => ({
    components: { Welcome },
    template: '<welcome></welcome>'
});

welcome.story = {
    parameters: { fileName: __filename, options: { showPanel: false, isToolshown: false } }
};
