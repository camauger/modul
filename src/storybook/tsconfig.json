{
    "compilerOptions": {
        "baseUrl": ".",
        "target": "es5",
        "esModuleInterop": true,
        "lib": [
            "es5",
            "es6",
            "dom"
        ],
        "module": "es2015",
        "moduleResolution": "node",
        "sourceMap": true,
        "experimentalDecorators": true,
        "allowSyntheticDefaultImports": true,
        "strictNullChecks": true,
        "paths": {
            "@ulaval/modul-components/dist/*": [
                "../../packages/modul-components/src/*"
            ],
            "@ulaval/modul-website/src/*": [
                "../modul-website/src/*"
            ]
        },
        "typeRoots": [
            "../../node_modules/@types"
        ],
        "plugins": [
            {
                "name": "typescript-tslint-plugin"
            }
        ]
    },
    "include": [
        "../../packages/modul-components/src/*",
        "../modul-website/src/*",
        "src/**/*.ts",
        "src/**/*.tsx",
        "src/**/*.vue",
        "tests/**/*.ts",
        "tests/**/*.tsx"
    ],
    "exclude": [
        "../../packages/modul-components/src/typings.d.ts",
        "../modul-website/src/typings.d.ts"
    ]
}
