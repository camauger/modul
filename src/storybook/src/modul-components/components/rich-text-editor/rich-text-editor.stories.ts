import { RICH_TEXT_EDITOR_NAME } from '@ulaval/modul-components/dist/components/component-names';
import { modulComponentsHierarchyRootSeparator } from '../../../../conf/storybook/utils';

export default {
    title: `${modulComponentsHierarchyRootSeparator}${RICH_TEXT_EDITOR_NAME}`
};

export const sandbox: any = () => '<m-rich-text-editor-sandbox></m-rich-text-editor-sandbox>';

sandbox.story = {
    parameters: { fileName: __filename, options: { showPanel: false, isToolshown: true } }
};
