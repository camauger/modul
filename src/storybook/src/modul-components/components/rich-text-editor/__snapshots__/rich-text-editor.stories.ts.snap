// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots modul-components|m-rich-text-editor Sandbox 1`] = `<m-rich-text-editor-sandbox />`;
